CONTRIBUTING.md

```diff
@@ -4,27 +4,46 @@ Thank you for contributing to Ersilia's mission and vision! ⭐
+- [Where to Start? 🧭](#where-to-start-🧭)
+- [How to Contribute 📜](#how-to-contribute-)
+- [Create a new Issue 🚀](#create-a-new-issue-)
+- [Contribution Workflow 🔄](#contribution-workflow-)
+- [Other Contributing Suggestions 💡](#other-contributing-suggestions-)
+- [I still have questions ❓](#i-still-have-questions-)
+
+## Code of Conduct 🤝
 
 We aspire to create a welcoming and safe environment for collaboration on this project and ask all contributors to do the same. Help us keep Ersilia open and inclusive. Please read and follow our [Code of Conduct](https://github.com/ersilia-os/ersilia/blob/master/CODE_OF_CONDUCT.md).
 
-## Where to Start?
+## Where to Start? 🧭
 
 The best place to start is by checking our issue tracker. Our issues have the following labels to guide contributors in their work:
 
-* good first issue: a small task suitable to start your contributions to the project
-* help wanted: an issue currently not assigned to anyone
-* bug: fix something that is not working
-* enhancement: add a new feature
-* documentation: issues related to the project's documentation
+| Label            | Description                                                     |
+|------------------|-----------------------------------------------------------------|
+| Good first issue | A small task suitable to start your contributions to the project|
+| Help wanted      | An issue currently not assigned to anyone                       |
+| Bug              | Fix something that is not working                               |
+| Enhancement      | Add a new feature                                               |
+| Documentation    | Issues related to the project's documentation                   |
+
+## How to Contribute 📜
 
 If this is the first time contributing to an Open Source project, read a bit about [How to Contribute to An Open Source Project on Github](https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github) and [Collaborating with pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests). If you land here during the Outreachy contribution period, please look for the appropriate guidelines in our [documentation](https://ersilia.gitbook.io/ersilia-book/contributors/internships).
 
 ## Create a new Issue 🚀
 
-The issue tracking avoids duplications and prevents different contributors working on the same feature separately, so please make sure to create a new issue with the appropriate labels before jumping on coding!
+The issue tracking avoids duplications and prevents different contributors working on the same feature separately, so please make sure to create a new issue with the appropriate labels before jumping on coding! 
+
+You can create the following issue types:
+* **🦠 Model Request:** Request to add a new model to the hub
+* **🐛 Bug Report:** Report something that is not working as expected
+* **📑 Feature Request:** Suggest a new feature or enhancement
```

Comment:
@adebisi4145 , you only listed 3 issue types?

Reply:
Thank you for the feedback @Malikbadmus . Initially, I focused on the 3 @DhanshreeA mentioned as the most important, but I will go ahead and add the others to make it more comprehensive.

Reply:
Hello @Malikbadmus , I have implemented the changes